frontend/packages/employee-frontend/src/components/messages/types.ts

```diff
@@ -21,3 +25,18 @@ export type IdAndName = {
   id: UUID
   name: string
 }
+
+export type SentBulletin = {
+  id: string
+  sentAt: Date
+  sender: string
+  title: string
+  content: string
+}
```

Comment:
If I remember correctly this does not include field `sender`, that is only returned to citizens at the moment. The only difference is that `sentAt` cannot be null. I believe this could be written shorter and drier as:
```
export type SentBulletin = Bulletin & {
  sentAt: Date
}
```